src/apps/expenses/components/CreateExpenseForm.js

```diff
@@ -509,10 +510,7 @@ class CreateExpenseForm extends React.Component {
     if (!LoggedInUser) {
       return (
         <div className="CreateExpenseForm">
-          <P textAlign="center" mt={4} fontSize="LeadParagraph" lineHeight="LeadParagraph">
-            <FormattedMessage id="expenses.create.login" defaultMessage="Sign up or login to submit an expense." />
-          </P>
-          <SignInForm next={`/${collective.slug}/expenses/new`} />
+          <SignInOrJoinFree redirect={Router.asPath} />
```

Comment:
@Betree This line triggers it. I am not sure if the Router instance is correctly defined.

Reply:
Try to omit this `redirect` parameter to see if the it works. The component fallsback to `window.location.pathname` so it should work without it.

Reply:
@Betree yes this resolves the above issue but the collective.test.js is still failing at this point.
![Screenshot_2019-06-19 Continuous Integration and Deployment](https://user-images.githubusercontent.com/31931348/59767195-63a9ec00-92bf-11e9-9dae-e4f6279603cd.png)
 
Look at this first error. I thought the above error was causing it. But seems like it is something else. Do you have any idea about what can be done for this because if it requests some element to get loaded and gets timed out before that then what changes can be made?

Reply:
The test was looking for a specific string and now the page has changed the tests are failing. This is expected.

The way to fix that is to run `npm run cypress:open` and to modify the tests or the code until they pass.

Reply:
For the first one above, the solution would be to modify the test to replace `Sign up or login to submit an expense` by a string that appears on the `SignInOrJoinFree` component.

Reply:
@Betree after debugging and making some changes in the collective.test.js the test passed successfully on my local machine but it is still failing over github.